src/modules/common/components/Selector.js

```diff
@@ -46,4 +61,4 @@ class Selector extends React.Component {
   }
 }
 
-export default Selector
+export default translate('common')(Selector)
```

Comment:
I think we shouldn't translate this component. `Selector` should be universally usable and not only with our I18n-library... Instead we should translate the tooltip one component above 🤔 I think it always makes sense to translate the translation-keys in that place where we spawn them :D

Reply:
(especially since the Selector is in `lib.js`)